docker-compose.yml

```diff
@@ -66,6 +66,8 @@ services:
             - scholarly-articles_fpm
     search_elasticsearch:
         image: docker.elastic.co/elasticsearch/elasticsearch:6.6.1
+        volumes:
+            - data-search:/usr/share/elasticsearch/data
```

Comment:
Could add `Z` modifier to indicate that the data is not shared
```suggestion
            - data-search:/usr/share/elasticsearch/data:Z
```

Reply:
What's the documentation for that? Should it be done here or at the time of volume creation?

Reply:
Here's the docs
https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label
I don't think its required at the time of creation, just for the mounts

Reply:
For context: ran into permissions issues running docker-compose on Ubuntu w/ SELinux enabled

Reply:
No adverse effects, adding.